testsuite/MDAnalysisTests/topology/test_hoomdxml.py

```diff
@@ -73,3 +73,10 @@ def test_dihedrals_identity(self, top):
         for b in ((0, 1, 2, 3), (1, 2, 3, 4), (2, 3, 4, 5), (3, 4, 5, 6)):
             assert (b in vals) or (b[::-1] in vals)
         assert ((0, 250, 350, 450) not in vals)
+
+    def test_read_masses(self, top):
+        for val in top.masses.values:
+            assert val == 1
+        assert ((0, -1, 250, 450) not in top.masses.values) 
+
+        
```

Comment:
Please add an equivalent test for charges as well. I understand that testing for 0 is a bit dumb but maybe we will change the test file in the future.